content/en/docs/concepts/policy/node-resource-managers.md

```diff
@@ -5,6 +5,7 @@ reviewers:
 title: Node Resource Managers 
 content_type: concept
 weight: 50
+#toc_hide: true
```

Comment:
Why comment it out? Consider setting this to `false` instead (or, omit the line).

Reply:
removed.